### DISCUSSION TITLE
Install and configuring KKP with Microsoft Azure

### DISCUSSION THREAD

**Discussion by udupicloud** [kubermatic/kubermatic#11060] [Q&A]
Hi everyone,

I am new to KKP and Azure. Need help in Installing and configuring KKP with Microsoft Azure.
I have configured K8s with Ceph on on-premise, which is working fine since a year. Now I want to move to Azure with KKP for hybrid solution.

I have tried the KKP CE installation links, failed to get the setup going. Can someone help me with the setup and steps that I need to follow to get it working. I am confused with the KKP document.

Thanks and Regards,
Mark.

**Comment by embik** [comment]
Hi Mark,

Is this is a duplication of #11078 or are you having questions outside of that other discussion? Can you tell us what about https://docs.kubermatic.com/kubermatic/v2.21/installation/install-kkp-ce/ specifically confuses you?
